aea/context/base.py

```diff
@@ -85,6 +86,12 @@ def __init__(
         self._default_routing = default_routing
         self._storage_callable = storage_callable
         self._namespace = SimpleNamespace(**kwargs)
+        self._build_dir = build_dir
+
+    @property
+    def build_dir(self) -> str:
+        """Return storage instance if enabled in AEA."""
+        return self._build_dir
```

Comment:
Why does this need to be exposed on context?

Reply:
thts correct, cane be reached by get_build_dir method